packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -317,12 +317,9 @@ export class PerfEntryRecorder {
   }
 
   start(type) {
-    /**
-     * Safari throws an error when PerformanceObserver is
-     * observed for unknown entry types as longtasks and lcp is
-     * not supported at the moment
-     */
-    try {
```

Comment:
I've removed try-catch based on the description in the comment, if folks think it's too risky, I will put it back.

Reply:
I would remove the comment but I wouldn't remove the try-catch.

- **Main reason**: It's a browser API, I would treat it as an external dependency,  then I would follow a defensive approach.
- The [spec]((https://www.w3.org/TR/performance-timeline/#observe-method)) mention "error" and "exception" multiple times. **Theoretically** the only that can affect us is the one about the invalid entry type, theoretically... 😄 
- There are a ton of browsers (and versions) out there, and it's not completely out of the box to see bugs like the one commented on this issue: https://github.com/elastic/sdh-kibana/issues/2547#issuecomment-1029054684

Reply:
Thanks Alberto, all make 💯 sense!! I've put back try-catch and added some comment inside catch 🙏